### DISCUSSION TITLE
unexpected token "?" when starting the dev server "npm run dev"

### DISCUSSION THREAD

**Discussion by BeshoyS** [vercel/next.js#45693] [Help] [answered]
Hey Everyone,
I have a problem when I use npx create-next-app, I have these warnings 
![image](https://user-images.githubusercontent.com/72396606/217507013-b822ff5f-ed1a-4807-ae70-86920c284a13.png)
![image](https://user-images.githubusercontent.com/72396606/217507208-59ff9575-80aa-4994-ab19-fbcad0a1b4f3.png)
![image](https://user-images.githubusercontent.com/72396606/217507324-4ebffe6e-34cb-438f-9bc6-e22fd672b592.png)

I have another problem when I start the dev server with create-next-app as shown below.

![image](https://user-images.githubusercontent.com/72396606/217507566-11d33e18-df2a-4d7f-81ea-477e70d092af.png)

I have npm: 9.4.0 and node 19.6.0 used.
I tried to reinstall both npm and node after deleting the npm-cache file, but I have the same problem.
please let me know how to fix these problems

**Comment by icyJoseph** [comment, marked as answer]
Ah this problem... let's see, there's a "core" node version installed on your system, and then there's the versions you've installed with `nvm`.  If you open the Windows cmd and run `node -v` what do you see? This is kind of why I left `nvm` behind, it makes this sort of thing very hard to debug. I can recommend `fnm` instead, but let's try to debug this.

**Reply by BeshoyS** [reply]
I uninstalled nvm and installed fnm and I have the same problem, I don't know what's wrong.

**Reply by icyJoseph** [reply]
uff I am sorry, but without access to that OS I don't think I can help much more other than asking you to check your PATH, or check what node version you have installed when you open CMD shell or Powersheel.

**Reply by BeshoyS** [reply]
Hi @icyJoseph,
I solved this problem by updating the system's node version. thanks for your help

**Comment by Qodestackr** [comment]
nvm use was not the issue for me. I just needed to do:
```bash
nvm install v18.12.1
nvm use v18.12.1
# Then Quits and Comes back to The Terminal (either on VS or normal). npm run dev should now work.
```
